src/main/java/org/xstefank/model/TyrProperties.java

```diff
@@ -0,0 +1,36 @@
+package org.xstefank.model;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import static org.xstefank.model.Utils.JBOSS_CONFIG_DIR;
+import static org.xstefank.model.Utils.CONFIG_FILE;
+
+public class TyrProperties {
+
+    private static final Properties properties =
+            loadProperties(System.getProperty(JBOSS_CONFIG_DIR), CONFIG_FILE);
+
+    public static String getStringProperty(String key) {
```

Comment:
also here `getProperty` should be sufficient enough right? user can see the return type